python/lsst/dax/apdb/scripts/metadata.py

```diff
@@ -97,3 +97,27 @@ def metadata_show(config: str, use_json: bool) -> None:
     else:
         for key, value in apdb.metadata.items():
             print(f"{key}: {value}")
+
+
+def check_instrument(instrument: str) -> None:
+    """Check that a fully-qualified instrument name is valid, if pipe_base is
+    available.
```

Comment:
I don't think this is the correct place for this code. `scripts/metadata.py` implements subcommands of `apdb-cli metadata`. Did you mean to put this in `Apdb`, `ApdbCassandra`, `ApdbMetadata` or `ApdbMetadataCassandra`?